### ISSUE TITLE
remove all string from code

### ISSUE THREAD

**Issue by chintans** [Promact/promact-oauth-server#144]
Currently, all strings are inside the code in a string constant. If I want to change any string, I would need to redeploy the application. Suggest a better way to manage text outside code and implement it.

**Comment by iamronakshah** [comment]
Start Date : 27-10-2016
Estimation : 5 Hours 
Finish Date : 28-10-2016

**Comment by iamronakshah** [comment]
I have reviewed the issue and found the solution. It is to create json file for those string which may change and rest must be same as it is in stringconstant.cs.  if anybody has any better solution then please suggest me.

**Comment by chintans** [comment]
Please submit the proposed schema of json file

**Comment by iamronakshah** [comment]
I have created sample json file which is content array of string constant. 
and this file is accessed (or read) in Respository , controller and testcase.cs (where stringcostant is used). 
but I have one question why are we changing the string because mostly all string are used in xunit test.

**Comment by chintans** [comment]
There are many strings used in code that will be deployed. So, If remove strings from the code changing them becomes easy.

**Comment by chintans** [comment]
> I have created sample json file which is content array of string constant. 

Please submit the schema of json file that you will be using.

**Comment by iamronakshah** [comment]
@chintans already I have mail you my sample json file.

**Comment by iamronakshah** [comment]
I Have uploaded sample json file.
[StringConstant.txt](https://github.com/Promact/promact-oauth-server/files/568804/StringConstant.txt)

**Comment by chintans** [comment]
Why don't you just use the coding highlighter and paste JSON in the comment instead of uploading a file?

**Comment by iamronakshah** [comment]
```
[
  {
    "DateFormate": "dd/MM/yyyy",
    "TeamLeaderNotAssign": "Not Assigned",
   "EmailNotExists": "Email does not exist",
    "ForgotPasswordTemplateFolderPath": "\\Template\\ForgotPassword.html",
    "ResetPasswordLink": "${{RestPassWordLink}}$",
    "ResertPasswordUserName": "${{Username}}$",
    "ForgotPassword": "Forgot Password",
    "SuccessfullySendMail": "We have sent you a link on {{emailaddress}} to reset password.Please check your email.",
    "InvalidLogin": "Invalid login attempt.",
    "SecretKeyGeneratorString": "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789",
    "FirstName": "First",
    "LastName": "Last",
    "RoleAdmin": "Admin",
    "RoleTeamLeader": "TeamLeader",
    "RoleEmployee": "Employee",
    "Format": "yyyy-MM-dd",
    "UserDetialTemplateFolderPath": "\\Template\\UserDetial.html",
    "UserEmail": "${{Email}}$",
    "UserPassword": "${{Password}}$",
    "LoginCredentials": "Login Credentials"
  }
]
```

**Comment by chintans** [comment]
This is not optimal format. We can't use this. Please improve schema for usability.

**Comment by iamronakshah** [comment]
I Have Improve my  schema  remove array now use like this.
```
{
    "DateFormate": "dd/MM/yyyy",
    "TeamLeaderNotAssign": "Not Assigned"
}
```

also we can use like this 
```
{
  "StringConstant": {
    "DateFormate": "dd/MM/yyyy",
     "TeamLeaderNotAssign": "Not Assigned"
    }
}
```

**Comment by chintans** [comment]
There are no usability improvements. It practically the same schema as before.

**Comment by iamronakshah** [comment]
@chintan 
Now question is that how to read json file in .net core?
In .net core we can use the IConfigurationRoot::Refresh() method to get the refresh value but the way in which they are reading the value is not optimal. because we can not assign the value to the model we have to use the string format.
So I have tried another way, i.e using IOptionsMonitor. It worked till rc2. It has been removed from .net core 1.0. Now .net core 1.1.0 has reintroduced that feature  in the use. so can I update .net core 1.0 to 1.1.0 prview 1.
